internal/pkg/mapper/defaultbucket/local_save.go

```diff
@@ -0,0 +1,50 @@
+package defaultbucket
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapBeforeLocalSave - replace default buckets in IM with placeholders.
+func (m *defaultBucketMapper) MapBeforeLocalSave(recipe *model.LocalSaveRecipe) error {
+	config, ok := recipe.Object.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	return m.visitStorageInputTables(config, m.replaceDefaultBucketWithPlaceholder)
+}
+
+func (m *defaultBucketMapper) replaceDefaultBucketWithPlaceholder(config *model.Config, sourceTableId string, inputTable *orderedmap.OrderedMap) error {
+	sourceConfigPath, found := m.getDefaultBucketSourceConfigurationPath(config, sourceTableId)
+	if !found {
+		return nil
+	}
+
+	tableName := strings.Split(sourceTableId, ".")[2]
```

Comment:
Tu by bolo lepsie asi `strings.SplitN(...., 3)`, kedze chceme vsetko za 2. bodkou.
Asi nemoze nastat, aby malo `tableId` > 2 `.`, ale je to teraz neosetrene ... a niekde moze byt aj nevalidna hodnota, ktora by sa tymto zmenila.

Reply:
ok 👍